pages/api/v3/packs/buy/[packType]/[uid].ts

```diff
@@ -90,9 +90,16 @@ const index = async (
 
     await portalQuery(
       SQL`
-          INSERT INTO bankTransactions (uid, status, type, description, amount, submitByID)
-          VALUES (${uid}, "completed", "cards", "Base Pack Purchase", -50000, 0);
-        `
+        INSERT INTO bankTransactions (uid, status, type, description, amount, submitByID)
+        VALUES (
+          ${uid}, 
+          "completed", 
+          "cards", 
+          ${packService.packs[packType].purchaseText},
+          ${-packService.packs[packType].price}, 
```

Comment:
Does this work? I would have guessed that the minus symbol needs to be outside the interpolation but I'm not certain at all

Reply:
In my testing it did.